src/Tizen.NUI.Scene3D/src/public/Controls/Camera.cs

```diff
@@ -67,9 +64,8 @@ public Camera() : this(Interop.Camera.New(), true)
         /// <summary>
         /// Copy constructor.
         /// </summary>
-        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
-        [EditorBrowsable(EditorBrowsableState.Never)]
-        public Camera(Camera copy) : this(Interop.Camera.NewCamera(Camera.getCPtr(copy)), true)
+        /// <since_tizen> 10 </since_tizen>
```

Comment:
`<param>` doc is missing, but please don't use word "handle" there, but rather sth like "The source Camera object", "The Camera object to be copied"

Reply:
Thank you I fixed.